src/install/multimc.md

```diff
@@ -6,18 +6,29 @@ permalink: /install/multimc/
 
 # MultiMC <a href="https://multimc.org" class="button is-link is-pulled-right"><span class="icon"><i class="fas fa-globe"></i></span><span>Website</span></a>
 
-### Status: Work In Progress
-
-MultiMC does not currently support Quilt, but a member of the community has 
-[submitted a pull request](https://github.com/MultiMC/Launcher/pull/3760) that adds Quilt support. If you're 
-interested in seeing Quilt support in MultiMC, feel free to take a look at that PR.
-
-# Manual Installation
-
-While it is technically possible to add Quilt to any MultiMC instance manually, it's a somewhat technical process that
-involves writing your own JSON component definition. For that reason, we're not publishing instructions on how to do 
-that here &mdash; however, we are working on automatically generating MultiMC instances you can import to easily get 
-set up with Quilt.
+{% admonition Development version required/danger %}
```

Comment:
Without the header title for consistency - just `/danger` is correct. That's a feature we never ended up using, haha - you can stick it in a h2 in the admonition itself instead.